gcp/appengine/frontend3/src/styles.scss

```diff
@@ -687,23 +687,49 @@ dl.vulnerability-details,
     }
   }
 
-
+  dd.credits {
+    ul {
+      padding: 0;
+    }
+    
+    li.credit {
+      margin: 0 0 10px 0;
+    }
+    
+    .contact li {
+      margin: 5px 40px 10px 0;
+      display: inline-flex;
+      flex-direction: row;
+      overflow-wrap: break-word;
+    }
+  }
+  
   pre {
     white-space: pre-wrap;
     overflow: auto;
   }
-
+  
+  .purl{
```

Comment:
can you add a space here after "purl" ?

Reply:
updated [511d314](https://github.com/google/osv.dev/commit/511d314f4238680672f70a2f1f3eff837b059c32)